privacy-policy.md

```diff
@@ -0,0 +1,34 @@
+# Privacy Notice
```

Comment:
No need to have it in root directory, tell me where to move it to - an entire new directory seems a bit much. Ok to just remove it as well